ci/build-docker.sh

```diff
@@ -93,13 +93,13 @@ if [ "${TEST_INSTALL}" = "yes" ]; then
   echo "${COLOR_YELLOW}Testing install rule.${COLOR_RESET}"
   cmake --build . --target install
   echo
-  echo "${COLOR_YELLOW}Test installed libraries using make(1).${COLOR_RESET}"
-  make -C /v/ci/test-install all
-  echo
   echo "${COLOR_YELLOW}Test installed libraries using cmake(1).${COLOR_RESET}"
-  cd /v/ci/test-install
-  CMAKE_PREFIX_PATH=/usr/local/share cmake -H. -B.build
-  cmake --build .build
+  cmake -H/v/ci/test-install -B/v/build-output/test-install-cmake
```

Comment:
As is done above, can you define your path in a variable and pass that as an arg to both the cmake commands? E.g. something like:

    readonly CMAKE_TEST_BUILD_DIR="/v/build-output/test-install-cmake"
    cmake -H/v/ci/test-install -B"${CMAKE_TEST_BUILD_DIR}"
    cmake --build "${CMAKE_TEST_BUILD_DIR}"

This would prevent someone from accidentally changing the path in one line but not the other.